src/app/forms/templates/test-records/section-templates/test/desk-based/desk-based-test-section-group3.template.ts

```diff
@@ -1,7 +1,7 @@
 import { ValidatorNames } from '@forms/models/validators.enum';
 import { FormNode, FormNodeEditTypes, FormNodeTypes, FormNodeViewTypes, FormNodeWidth } from '@forms/services/dynamic-form.types';
 
-export const DeskBasedTestSectionGroup3Psv: FormNode = {
+export const DeskBasedTestSectionGroup3: FormNode = {
```

Comment:
I'd prefer if the Psv suffix stayed on just for understanding the purpose of the template.

Reply:
wait...